src/ts/track2-mgmt-sdk-release-tools/Readme.md

```diff
@@ -0,0 +1,144 @@
+# Azure Track2 JS SDK Tools
```

Comment:
Please put this under "tools/mgmt-sdk-release-tools" directory. I'd like to avoid having the name track2 baked into the tools and directories as that is a temporary point in time name.

Reply:
I move it to tools/js-mgmt-sdk-release-tools because I think it is only for js sdk